tensorboard/webapp/hparams/_redux/types.ts

```diff
@@ -43,6 +45,8 @@ export const HPARAMS_FEATURE_KEY = 'hparams';
 
 export interface HparamsState {
   specs: ExperimentToHparams;
```

Comment:
Nit: To avoid confusion I think this should be name experimentListSpecs or something like that.

Reply:
I agree but I'd like to refactor the experiment list portion of the state in a separate PR once the dashboard flow is fully checked in. I think this new flow is better and I am hopeful that we can remove a lot of the old code.